### ISSUE TITLE
Biligual Support

### ISSUE THREAD

**Issue by tpcolson** [nationalparkservice/npmap-builder#269]
Getting a few requests that builder maps on the park CMS web page offer bilingual support. E.g. if we publish a map that is titled "Restrooms", there should be an option for the title to be rendered in Spanish. Alt text for icons should also support bilingual rendering.
